product/roundhouse.tasks/Roundhouse.cs

```diff
@@ -30,7 +41,7 @@ public sealed class Roundhouse : ITask, ConfigurationPropertyHolder
         bool ITask.Execute()
         {
             run_the_task();
-            return true;
+            return !this.loggingHelper.HasLoggedErrors;

```

Comment:
Getting to this may be harder without direct access to the logger. It should be possible, just not quite as straightforward.